spec/models/post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+RSpec.describe Post, type: :model do
+  # tests go here
+  it 'is not valid without a title' do
+    post = Post.new(title: nil)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid if title length is more that 250 characters' do
+    post = Post.new(title: 'a' * 251)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid if comments_counter is less than 0' do
+    post = Post.new(comments_counter: -1)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid if likes_counter is less than 0' do
+    post = Post.new(likes_counter: -1)
+    expect(post).to_not be_valid
+  end

```

Comment:
- [ ] Kindly add some positive test cases also.

Reply:
Thank you very much. Indeed it is a good idea to have both negative and positive test cases in test files. I will make the change immediately